scripts/paternity-test.py

```diff
@@ -0,0 +1,173 @@
+import io
+import os
+import json # might not be needed
+from firecloud import fiss
+from firecloud.errors import FireCloudServerError
+import firecloud.api as fapi
+import numpy as np
+import pandas as pd
+import logging
+#import re <-- doesn't work on Terra
+
+# User-set variables
+BILLING_PROJECT_ID="biodata-catalyst"
+WORKSPACE="TSV-AFY"
+SUBDIRECTORY="\/case2\/"
+TABLE_NAME="CRAMs_and_CRAIs" #Do not include spaces or weird characters
+PARENT_FILETYPE="cram"
+# If your filenames are like this, please set INCLUDE_PARENT_EXTENSION to True:
+# NWD119844.CRAM
+# NWD119844.CRAM.CRAI
+# If you filenames are like this, please set INCLUDE_PARENT_EXTENSION to False:
+# NWD119844.CRAM
+# NWD119844.CRAI
+# No quotation marks for this variable, just True or False
+INCLUDE_PARENT_EXTENSION = True
+# Make sure to escape the slashes in your SUBDIRECTORY variable. 
+# For instance, if your files are in a folder called "testfiles" you will need to
+# enter it as "/\testfiles/\" or else Python will get angry.
+
+# Call FireCloud API
+try:
+    bucket = os.environ["WORKSPACE_BUCKET"]
+    response = fapi.list_entity_types(BILLING_PROJECT_ID, WORKSPACE)
+    if response.status_code != 200:
+        print("Error in Firecloud, check your billing project ID and the name of your workspace.")
+    else:
+        print("Firecloud has found your workspace!")
+        directory = bucket + SUBDIRECTORY
```

Comment:
"directory" is a bit of a misnomer, since buckets are not filesystems. I'd generally use "prefix" or "key_prefix" instead.